coc/players.py

```diff
@@ -621,20 +625,18 @@ def heroes(self) -> typing.List[Hero]:
         self._heroes = sorted_heroes
         return list(sorted_heroes.values())
 
-    def get_hero(self, name: str, default_value=None) -> typing.Optional[Hero]:
```

Comment:
Any reason why this docstring was changed so much? I think the one before is more descriptive :')